### PR TITLE
docs(295): update readme w/ Stripe, Segment docs, Production network name

### LINKED ISSUE #280: Update docs on proper production network name
Once #277 lands, we'll need to revisit the README.md documentation about running with a production network name. We don't want users running the `*.reaction.localhost` network name in production. This issue is a TODO to follow through with that.

### LINKED ISSUE #295: Docs: Update README
- [ ] Add a screenshot at the top
- [x] Add Stripe env instructions
- [x] Add instructions/linking back to Component Library and Reaction
- [ ] Add description and website copy at the top
- [x] Decide whether to link to the staging/test url or not
- [x] Make it clear in the README whether this repository is open to new features, or whether new feature requests should go into: https://github.com/reactioncommerce/reaction-feature-requests
- [x] Make it clear in the README what the development status of this project is

### LINKED ISSUE #296: 📋 Open source checklist
In order to prep this repository for open source use and consumption, we need to complete this checklist:

## Repository checklist
- [x] Add `.github` directory with pull request template, issue template(s) (1 template for bugs, 1 template for features)
- [x] Add issue template(s): bugs, features
- [x] Add Reaction's standard `LICENSE.md`
- [x] Add Reaction's standard `CODE_OF_CONDUCT.md`
- [x] Add Reaction's standard `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by machikoyasuda**
Resolves #295 
Resolves #296 
Resolves #280 
Impact: **minor**
Type: **docs|chore**

## Issue/Solution
- README now explains how to set up Stripe, does not link to Segment setup
- README now adds links back to Component Library and other relevant docs
- Add CONTRIBUTING, LICENSE (GNU), CODE OF CONDUCT docs -- all from the main Reaction repo
- Add bug ticket template

## Testing
1. Test Stripe API key configuration steps

### CODE REVIEW

**Comment by aldeed** [comment]
@machikoyasuda Can you address #280 in this PR, too? Essentially we want to clarify that the "Production" section is about how to build and run the production app locally, not a recommendation for production deployment. Maybe change that section to something like this:

## ~~Production~~ Building and running the production app locally
~~Running the command below will build the starterkit for production.~~
Sometimes it's helpful during development to make a production build of the app and run that. Run this command to build a Docker image with the production build of the app in it:

```
docker build -t reaction-storefront --build-arg BUILD_ENV=production .
```

~~To start the app in production mode execute:~~
Then to start the app on your machine, make sure the Reaction API container is already running and enter:

```
docker run -d --name storefront -p ${port}:4000 --env-file .env --network api.reaction.localhost reaction-storefront
```

**NOTE:** Replace the `${port}` with the localhost port you'd like the application to run at
**NOTE:** This is not the way to run the app in an actual production deployment. This is only for running the production build locally for development, demo, or trial purposes.

To stop the docker container after starting it with the above command
```
docker stop storefront
```

**Comment by machikoyasuda** [comment]
@aldeed added 👍

File: README.md [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
-Reference Storefront application for [Reaction Commerce](https://reactioncommerce.com/).
+Reference headless ecommerce storefront starter kit application for [Reaction Commerce](https://reactioncommerce.com/), featuring [React](https://reactjs.org/), [Next.js](https://nextjs.org/) and [Apollo](https://www.apollographql.com/docs/react/) to connect to the [Reaction GraphQL API](https://github.com/reactioncommerce/reaction/), available as of Reaction v2.0.0.
+
+## Features
+
+- Headless ecommerce starter kit with [Next.js](https://nextjs.org/), [React](https://reactjs.org/), [MobX](https://mobx.js.org/getting-started.html), [GraphQL](https://graphql.org/), [Apollo](https://www.apollographql.com/docs/react/)
+- Server-side rendering
+- Payments with [Stripe](https://stripe.com/)
+- Analytics with [Segment](https://segment.com/)
+- Reusable, customizable, themeable ecommerce React components from the new [Reaction Component Library](https://www.npmjs.com/package/@reactioncommerce/components) with [Styled Components](https://www.styled-components.com/)
```
Comment (willopez) [inline README.md]: The link to the component library is not working, lets just link to the github repo

File: README.md [hunk 2]
```diff
@@ -1,22 +1,34 @@
...
+- Fully-configured test suite: Jest snapshot testing, Mocha integration testing
+- Written in ES6, configured with ES6
+- Containerized with Docker
 
 ## Getting Started
```
Comment (willopez) [inline README.md]: Lets have people start using this storefront by using the reaction platform, which will simplify things and remove most of this manual setup process. Plus, that is why the reaction platform project exists.

**Comment by willopez** [comment]
@machikoyasuda this section "Generate a Meteor login token" can be removed, as authentication is implemented. Further, using the reaction platform will automatically copy the .env.example inte .env, so that section can be removed also.

**Comment by willopez** [comment]
running the platform will give you good idea of the current best way to run reaction and all its sub-projects.

**Comment by machikoyasuda** [comment]
Ready for re-review @willopez

**Comment by machikoyasuda** [comment]
@spencern If you have time -- can you review this PR for the changes to License.md, Code of Conduct and Contributing docs? And the general description? The license, coc and contributing docs are all straight from Reaction.

File: README.md [hunk 3]
```diff
@@ -1,76 +1,88 @@
...
-5. Generate a Meteor login token
+## Configuration
 
-    _This process will be eliminated once we've built out the GraphQL API for authentication_
-    - Visit the Reaction Meteor shop [localhost:3000](http://localhost:3000)
-    - Open devtools and copy the Meteor.loginToken from `localStorage`.
+### Set up Stripe
 
-6. Setup the Storefront environment
+When running the storefront from a new Reaction shop, you will need to configure Stripe payment processing and shipping options to test a complete order checkout flow. After signing up for a Stripe API key, follow these steps:
```
Comment (willopez) [inline README.md]: lets change: "When running the storefront from a new Reaction shop" to "When running the storefront and Reaction for the first time,"

**Comment by machikoyasuda** [comment]
@willopez Ready for re-review!

**Comment by willopez** [comment]
@machikoyasuda there are some comments that have not been addressed.

**Pull request merged**